doc/logos.md

```diff
@@ -40,7 +40,7 @@ If you are submitting a pull request for a custom logo, please:
     - Install SVGO
       - With npm: `npm install -g svgo`
       - With Homebrew: `brew install svgo`
-    - Run the following command `svgo --precision=3 icon.svg icon.min.svg`
+    - Run the following command `svgo --precision=3 icon.svg`
```

Comment:
Could be. But why do you want to use `.min`? Existing filename has `.svg`, not `.min.svg`. A logo contributor needs to rename `.min.svg` to `.svg` after minification if your suggestions applied.

Reply:
Right, we're not concerned with the actual file name @tnir as this is just an example. It could be `foo.svg -o bar.svg` for all it matters. The `*.min.svg` naming pattern is what svgo happen to use as the example file names in their own cli docs, so I don't think it's a big deal either way

Reply:
I prefer `svgo --precision=3 icon.svg`, which is equivalent to `svgo --precision=3 icon.svg -o icon.svg`, but I would like to get this merged sooner. I pushed the changes @chris48s suggested.

Reply:
Forgive me as I'm not intimately familiar with svgo's command line surface, but isn't what you're suggesting @tnir overwriting the original file? I believe we're specifically trying to avoid having instructions that result in modification to the original in case users aren't terrible familiar and would like to keep the original.

Reply:
@calebcartwright That makes sense to me. I hope the current change in the PR gets merged soon.